core/input/fixedkeys.js

```diff
@@ -0,0 +1,111 @@
+/*
+ * noVNC: HTML5 VNC client
+ * Copyright (C) 2017 Pierre Ossman for Cendio AB
+ * Licensed under MPL 2.0 or any later version (see LICENSE.txt)
+ */
+
+/* [module]
+ * import KeyTable from "./keysym";
+ */
+
+"use strict";
+
+var fixedkeys = {
```

Comment:
I have a *huge* table of these in my PR from a while back that I did the research on.  We may just want to include that, since it's a bit more expansive, IIRC.